pkg/config/raw_image_spec.go

```diff
@@ -0,0 +1,77 @@
+package config
+
+type rawImageSpec struct {
+	Author        string             `yaml:"author,omitempty"`
+	RemoveHistory bool               `yaml:"removeHistory,omitempty"`
+	Config        rawImageSpecConfig `yaml:"config,omitempty"`
+
+	rawStapelImage        *rawStapelImage        `yaml:"-"` // parent
+	UnsupportedAttributes map[string]interface{} `yaml:",inline"`
+}
+
+type rawImageSpecConfig struct {
+	RemoveWerfLabels bool     `yaml:"removeWerfLabels,omitempty"`
+	RemoveLabels     []string `yaml:"removeLabels,omitempty"`
+	RemoveVolumes    []string `yaml:"removeVolumes,omitempty"`
+	RemoveEnv        []string `yaml:"removeEnv,omitempty"`
+
+	Volumes     []string          `yaml:"volumes,omitempty"`
+	Labels      map[string]string `yaml:"labels,omitempty"`
+	Env         map[string]string `yaml:"env,omitempty"`
+	Expose      []string          `yaml:"expose,omitempty"`
+	User        string            `yaml:"user,omitempty"`
+	Cmd         []string          `yaml:"cmd,omitempty"`
+	Entrypoint  []string          `yaml:"entrypoint,omitempty"`
+	WorkingDir  string            `yaml:"workingDir,omitempty"`
+	StopSignal  string            `yaml:"stopSignal,omitempty"`
+	Healthcheck *healthConfig     `yaml:"healthcheck,omitempty"`
+
+	UnsupportedAttributes map[string]interface{} `yaml:",inline"`
```

Comment:
Unsupported attributes are not handled the same way as elsewhere, and you can specify any directive without triggering an error